practical_07/project_management.py

```diff
@@ -0,0 +1,207 @@
+"""
+
+Estimated completion time: 3h
+Actual completion time:
+"""
+
+from project import Project
+from datetime import datetime
+from operator import attrgetter
+
+NAME_INDEX = 0
+START_DATE_INDEX = 1
+PRIORITY_INDEX = 2
+COST_ESTIMATE_INDEX = 3
+COMPLETION_PERCENTAGE_INDEX = 4
+
+MENU = """
+- (L)oad projects  
+- (S)ave projects  
+- (D)isplay projects  
+- (F)ilter projects by date
+- (A)dd new project  
+- (U)pdate project
+- (Q)uit
+
+>>> """
+
+
+def main():
+    menu_choice = input(MENU).upper()
```

Comment:
Maybe load your original projects here? What would happen if "L" wasn't the first option?